oak_functions_launcher/README.md

```diff
@@ -1,29 +1,29 @@
 # Oak Functions Launcher
 
-Untrusted companion app that loads the Oak Functions binary in either QEMU or
-crosvm, and exposes a gRPC server for communicating with the binary.
-Communicates with the binary via the `oak_channel`.
+The Oak Functions Launcher loads the Oak Functions enclave binary and exposes a
+gRPC server for communicating with the binary via the
+[Oak Channel](../oak_channel).
 
-## Starting the binary
+## Launching the Oak Functions enclave binary
 
-The Oak Functions binary may be loaded in crosvm via
+First, set the permission to access the KVM kernel module by either (i)
+`sudo setfacl -m u:${USER}:rw /dev/kvm` on the host, or by (ii) adding the user
```

Comment:
It might be worth noting that this is only needed if "rootless" Docker (as suggested in "develpoment.md") is used. For rootfull Docker (the default Docker mode) this is not needed.

Reply:
True, thanks, added!